cmd/packages.go

```diff
@@ -186,9 +185,13 @@ func bindPackagesConfigOptions(flags *pflag.FlagSet) error {
 }
 
 func packagesExec(_ *cobra.Command, args []string) error {
-	errs := packagesExecWorker(args[0])
-	ux := ui.Select(appConfig.CliOptions.Verbosity > 0, appConfig.Quiet)
-	return ux(errs, eventSubscription)
+	return eventLoop(
+		packagesExecWorker(args[0]),
```

Comment:
nit: If we're passing in a specific arg, it could be a nice hint to the reader to identify the arg via an explicit parameter for `packagesExec`, rather than receive the whole slice and grab item `0` just in time.